app/views/fact_values/_fact.html.erb

```diff
@@ -13,11 +13,11 @@
   </td>
   <td class="fact-col">
     <% unless fact.compose %>
-      <%= link_to trunc_with_tooltip(fact.value, 40),
+      <%= link_to truncate(fact.value, :length => 60),
```

Comment:
interesting, my characters seemed to be less wide :-) ok, setting 50 (even though I guess 50 of `@` would overflow as well